webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/render/BratRenderer.java

```diff
@@ -294,7 +294,7 @@ public static void renderTokens(CAS aCas, GetDocumentResponse aResponse, Annotat
                             aResponse.addEntity(new Entity(new VID(fs), "Token",
                                     new Offsets(range.getBegin(), range.getEnd()),
                                     fs.getCoveredText(), "#d9d9d9",
-                                    "[" + fs.getBegin() + "-" + fs.getEnd() + "]"));
+                                    "[" + fs.getBegin() + "-" + fs.getEnd() + "]", null));
```

Comment:
IMHO the "buttons" flag should be `false` here and not `null`.

Reply:
Well I thought about putting it to `false` at first, but as you mentioned earlier that during the serialization of the properties those that are not set (as i understood  `null`) will not be added to the Entity which saves bandwidth. Have I understood correctly?

Reply:
You understood correctly that we wish to save bandwidth. But it does not necessarily mean that we cannot still have a reasonable API. Jackson usually sets its values through getters/setters, so we can do in the constructor what we want. E.g. we can have a boolean argument and internally convert it to a number, string, whatever. Also, Jackson seems to have some provisions for rendering boolean fields as 0/1 in the JSON and for omitting primitive values which have the default (see stackoverflow links in the other comment). So we might even get away with having a `boolean` field internally and still have minimal bandwidth impact.